rs/execution_environment/src/canister_manager/tests.rs

```diff
@@ -4765,124 +4118,6 @@ fn system_subnet_does_not_check_for_freezing_threshold_on_allocation_changes() {
         .unwrap();
     test.canister_update_allocations_settings(canister_id, Some(0), Some(0))
         .unwrap();
-
-    let canister_id = test.create_canister(Cycles::new(1_000_000_000_000));
-    test.install_canister_with_allocation(
-        canister_id,
-        UNIVERSAL_CANISTER_WASM.to_vec(),
-        Some(50),
-        None,
-    )
-    .unwrap();
-    test.canister_update_allocations_settings(canister_id, Some(0), Some(0))
-        .unwrap();
-
-    let canister_id = test.create_canister(Cycles::new(1_000_000_000_000));
-    test.install_canister_with_allocation(
-        canister_id,
-        UNIVERSAL_CANISTER_WASM.to_vec(),
-        None,
-        Some(10 * 1024 * 1024 * 1024),
-    )
-    .unwrap();
-    test.canister_update_allocations_settings(canister_id, Some(0), Some(0))
-        .unwrap();
-}
```

Comment:
These paths are now irrelevant for this test (system_subnet_does_not_check_for_freezing_threshold_on_allocation_changes). The allocation changes are checked above via the `update_settings` path instead of using `install_code`.